src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml

```diff
@@ -121,8 +121,7 @@
                 <Setter Property="Template">
                     <Setter.Value>
                         <ControlTemplate TargetType="{x:Type TreeViewItem}">
-                            <Button Command="{Binding ClickedCommand}"
-                                    Focusable="False"
```

Comment:
If we remove `Focusable="False"`, then button can be in focus. That means search box will lose focus and we don't want it. To try it do next:
1. Search for some word, e.g. "faces".
2. Click on "ExtractSolids".
3. Notice, that focus from search box is lost.